sdk-actors/src/main/java/io/dapr/actors/client/DaprHttpClient.java

```diff
@@ -46,9 +36,9 @@
    */
   @Override
   public Mono<byte[]> invoke(String actorType, String actorId, String methodName, byte[] jsonPayload) {
-    String url = String.format(ACTOR_METHOD_RELATIVE_URL_FORMAT, actorType, actorId, methodName);
+    String[] path = new String[] { DaprHttp.API_VERSION, "actors", actorType, actorId, "method", methodName };
     Mono<DaprHttp.Response> responseMono =
-          this.client.invokeApi(DaprHttp.HttpMethods.POST.name(), url, null, jsonPayload, null, null);
+          this.client.invokeApi(DaprHttp.HttpMethods.POST.name(), path, null, jsonPayload, null, null);
```

Comment:
Is there a test for the case that's being fixed? (Reserved URL Path) character in this state store or method name?

Reply:
Will do.